internal/msalbase/account_test.go

```diff
@@ -8,17 +8,19 @@ import (
 	"testing"
 )
 
-var accHID = "hid"
-var accEnv = "env"
-var accRealm = "realm"
-var authType = "MSSTS"
-var accLid = "lid"
-var accUser = "user"
+var (
+	accHID   = "hid"
+	accEnv   = "env"
```

Comment:
nit: I would just write out `accHomeAccountId = 'homeAccountId"

Reply:
I would table this for the future.  These names are unchanged from the current dev code.  The tests need a lot of help and it is very likely that none of these will survive at the end.  acc prefix here  isn't useful and in the end, we probably are going to change how this is done.